docs/source/matchers.rst

```diff
@@ -142,6 +154,51 @@ Testing whether the transfer changes balance for multiple accounts:
   await expect(() => token.transfer(walletTo.address, 200))
     .to.changeTokenBalances(token, [walletFrom, walletTo], [-200, 200]);
 
+Change balance
+--------------
+.. deprecated:: 3.1.2
+   Use :func:`changeEtherBalance` instead.
+
+Testing whether the transaction changes the balance of the account:
+
+.. code-block:: ts
+
+  await expect(() => wallet.sendTransaction({to: walletTo.address, gasPrice: 0, value: 200}))
+    .to.changeBalance(walletTo, 200);
+
+  await expect(await wallet.sendTransaction({to: walletTo.address, gasPrice: 0, value: 200}))
+    .to.changeBalance(walletTo, 200);
+
+:code:`expect` for :code:`changeBalance` gets one of the following parameters:
+
+  - **transaction call** : () => Promise<`TransactionResponse <https://docs.ethers.io/v5/api/providers/types/#providers-TransactionResponse>`_> - we first check the balance then call the transaction callback and finally calculate the difference between current balance and the balance before the transaction.
+  - **transaction response** : `TransactionResponse <https://docs.ethers.io/v5/api/providers/types/#providers-TransactionResponse>`_ - we check the balance difference between the block that transaction was mined in and the block before it.
+
+.. note:: :code:`changeBalance` calls will not work unless there is only one transaction mined in the block.
+
+The transaction call should be passed to the :code:`expect` as a callback (we need to check the balance before the call) or as a transaction response.
+
+The matcher can accept numbers, strings and BigNumbers as a balance change, while the address should be specified as a wallet or a contract.
+
+.. note:: :code:`changeBalance` calls should not be chained. If you need to chain it, you probably want to use :code:`changeBalances` matcher.
+
+Change balance (multiple accounts)
+----------------------------------
+.. deprecated:: 3.1.2
+   Use :func:`changeEtherBalances` instead.
+
+Testing whether the transaction changes balance for multiple accounts:
+
+.. code-block:: ts
+
+  await expect(() => wallet.sendTransaction({to: walletTo.address, gasPrice: 0, value: 200}))
+    .to.changeBalances([walletFrom, walletTo], [-200, 200]);
+
+  await expect(await wallet.sendTransaction({to: walletTo.address, gasPrice: 0, value: 200}))
+    .to.changeBalances([walletFrom, walletTo], [-200, 200]);
+
+.. note:: :code:`changeBalances` calls will not work unless there is only one transaction mined in the block.
+
```

Comment:
Let's add a `Deprecated matchers` section at the bottom of the document and move these two matchers there. Also please apply my suggestions form `changeEtherBalance(s)` here as well to make the doc consistent.